tools/single_use/populate_scan_type_identification_failure_table_based_on_mri_protocol_violated_scans_table.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * This script is written for a one time use only to insert into the
+ * scan_type_identification_failure table entries based on what is present in
+ * the mri_protocol_violated_scans table.
+ *
+ * Affected table: scan_type_identification_failure
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Loris Team <loris.mni@bic.mni.mcgill.ca>
+ * @license  Loris License
+ * @link     https://github.com/aces/Loris
+ */
+require_once __DIR__ . '/../generic_includes.php';
+require_once 'Database.class.inc';
```

Comment:
This should be autoloaded

Reply:
Do you mean that it should be autoloaded because of the `require_once generic_includes.php`? Just making sure I understand correctly. Thank you!

Reply:
I wasn't sure, so I just double checked and it includes

```
require_once __DIR__ . "/../vendor/autoload.php";
```

So, yes, generic_includes should already load the autoloader.

Reply:
ok. Will be done in the next push.